src/domain/collaboration/InnovationPack/InnovationPackCardHorizontal/InnovationPackCardHorizontal.tsx

```diff
@@ -0,0 +1,121 @@
+import DesignServicesIcon from '@mui/icons-material/DesignServices';
+import { Box, Skeleton } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import CardFooterCountWithBadge from '../../../../core/ui/card/CardFooterCountWithBadge';
+import { gutters } from '../../../../core/ui/grid/utils';
+import RouterLink from '../../../../core/ui/link/RouterLink';
+import BadgeCardView from '../../../../core/ui/list/BadgeCardView';
+import { Caption, CardTitle } from '../../../../core/ui/typography';
+import { CommunityGuidelinesIcon } from '../../communityGuidelines/icon/CommunityGuidelinesIcon';
+import { InnovationFlowIcon } from '../../InnovationFlow/InnovationFlowIcon/InnovationFlowIcon';
+import { PostIcon } from '../../post/icon/PostIcon';
+import { WhiteboardIcon } from '../../whiteboard/icon/WhiteboardIcon';
+import InnovationPackIcon from '../InnovationPackIcon';
+import OneLineMarkdown from '../../../../core/ui/markdown/OneLineMarkdown';
+import RoundedIcon from '../../../../core/ui/icon/RoundedIcon';
+
+export interface InnovationPackCardProps {
+  profile: {
+    displayName: string;
+    description?: string;
+    url: string;
+  };
+  calloutTemplatesCount?: number;
+  communityGuidelinesTemplatesCount?: number;
+  innovationFlowTemplatesCount?: number;
+  postTemplatesCount?: number;
+  whiteboardTemplatesCount?: number;
+}
+
+export const InnovationPackCardHorizontalSkeleton = () => (
```

Comment:
InnovationPackCardHorizontalSkeleton && InnovationHubCardHorizontalSkeleton looks the same, can we use just one of them for both skeletons?

Reply:
I'd like to keep them separated in case their visualizations divert more, and together with their non-skeleton card... But I have done now the InnovationPack skeleton a bit different to have the excuse :D